Course_Part_03/src/Assignment11.java

```diff
@@ -1,6 +1,7 @@
 public class Assignment11 {
```

Comment:
een CTRL + ALT + L (code formatting zou hier nog kunnen). De reden om een for-each te gebruiken ten opzichte van een traditionele for met index is omdat je geen index nodig hebt.